algorithm/coordinate_compression/coordinate_compression.cpp

```diff
@@ -0,0 +1,41 @@
+#include "../../base.hpp"
+#include <vector>
+
+template <typename T1, typename T2>
+vector<pair<long long, long long>> coordinate_compression(vector<pair<T1, T2>> &coordinate)
```

Comment:
```suggestion
vector<pair<long long, long long>> coordinate_compression(const vector<pair<T1, T2>> &coordinate)
```
const をつけると、`coordinate_compression(vector<pair<int,int>>(10, pair(1,1)));` みたいにアドレスを持たない値も呼び出せます